clouds/snowflake/modules/sql/h3/H3_KRING_DISTANCES.sql

```diff
@@ -1,38 +1,28 @@
 ----------------------------
--- Copyright (C) 2021 CARTO
+-- Copyright (C) 2021-2024 CARTO
 ----------------------------
 
-CREATE OR REPLACE FUNCTION @@SF_SCHEMA@@._H3_KRING_DISTANCES
-(origin STRING, size DOUBLE)
+CREATE OR REPLACE SECURE FUNCTION @@SF_SCHEMA@@._H3_KRING_DISTANCES
+(origin STRING, hexarray ARRAY)
```

Comment:
Sam about naming.

In this function I don't know why added `SECURE` and also I think it should keep being `IMMUTABLE` as the results should be deterministic.